crates/std_detect/src/detect/arch/aarch64.rs

```diff
@@ -244,6 +245,7 @@ features! {
     @FEATURE: #[stable(feature = "simd_aarch64", since = "1.60.0")] sve2_aes: "sve2-aes";
     /// FEAT_SVE_AES & FEAT_SVE_PMULL128 (SVE2 AES crypto)
     @FEATURE: #[unstable(feature = "stdarch_aarch64_feature_detection", issue = "127764")] sve_b16b16: "sve-b16b16";
```

Comment:
This is actually a valid target feature, but it is only exposed on LLVM 20 (we are currently on LLVM 19). See compiler/rustc_codegen_llvm/src/llvm_util.rs